src/gui/dialogs/story_viewer.cpp

```diff
@@ -92,6 +92,8 @@ void story_viewer::pre_show()
 		std::bind(&story_viewer::nav_button_callback, this, DIR_FORWARD));
 	connect_signal_mouse_left_click(find_widget<button>("prev"),
 		std::bind(&story_viewer::nav_button_callback, this, DIR_BACKWARDS));
+	connect_signal<event::BACK_BUTTON_CLICK>(std::bind(&story_viewer::nav_button_callback, this, DIR_BACKWARDS), front_pre_child);
```

Comment:
For consistency the queue positions should be qualified with `event::dispatcher::`. Also, I'd like to move away from std::bind when possible, so suggest (and likewise with the other new handlers):
```cpp
connect_signal<event::BACK_BUTTON_CLICK>(
	[this](auto&&...) { nav_button_callback(DIR_BACKWARDS); }, event::dispatcher::front_pre_child);